tox.ini

```diff
@@ -221,31 +219,28 @@ setenv=
   container_podman: PLAYBOOK = site-docker.yml.sample
 
   rhcs: CEPH_STABLE_RELEASE = luminous
-  luminous: CEPH_STABLE_RELEASE = luminous
-  luminous: CEPH_DOCKER_IMAGE_TAG = latest-luminous
-  luminous: CEPH_DOCKER_IMAGE_TAG_BIS = latest-bis-luminous
-  luminous: UPDATE_CEPH_STABLE_RELEASE = luminous
-  luminous: UPDATE_CEPH_DOCKER_IMAGE_TAG = latest
-  mimic: CEPH_STABLE_RELEASE = mimic
-  mimic: CEPH_DOCKER_IMAGE_TAG = latest-mimic
-  mimic: CEPH_DOCKER_IMAGE_TAG_BIS = latest-bis
-  mimic: UPDATE_CEPH_STABLE_RELEASE = mimic
-  mimic: UPDATE_CEPH_DOCKER_IMAGE_TAG = latest
   lvm_osds: CEPH_STABLE_RELEASE = luminous
   lvm_osds: PLAYBOOK = site.yml.sample
-  lvm_osds_container: CEPH_STABLE_RELEASE = luminous
   lvm_osds_container: PLAYBOOK = site-docker.yml.sample
-  bluestore_lvm_osds: CEPH_STABLE_RELEASE = luminous
-  bluestore_lvm_osds_container: CEPH_STABLE_RELEASE = luminous
   bluestore_lvm_osds_container: PLAYBOOK = site-docker.yml.sample
   lvm_batch_container: PLAYBOOK = site-docker.yml.sample
-  lvm_batch_container: CEPH_STABLE_RELEASE = luminous
+  shrink_osd_container: COPY_ADMIN_KEY = True
+  shrink_osd: COPY_ADMIN_KEY = True
+
+  rhcs: CEPH_STABLE_RELEASE = luminous
+  dev: CEPH_STABLE_RELEASE = nautilus
+  dev: CEPH_DOCKER_IMAGE_TAG = latest-master
+  dev: CEPH_DOCKER_IMAGE_TAG_BIS = latest-bis
+  dev: UPDATE_CEPH_STABLE_RELEASE = nautilus
```

Comment:
Not sure if there is no collision with this https://github.com/ceph/ceph-ansible/pull/3364/files#diff-b91f3d5bd63fcd17221b267e851608e8R280